store.css

```diff
@@ -0,0 +1,157 @@
+:root {
+  --orange-dark: #de6e49;
```

Comment:
Great use of CSS variables here, keep doing the same for all colors in the CSS style.

Reply:
Thank you Alberto.